infrastructure/main.tf

```diff
@@ -118,6 +118,10 @@ data "vault_generic_secret" "puiWebapp" {
   path = "secret/${var.vault_section}/ccidam/service-auth-provider/api/microservice-keys/pui-webapp"
 }
 
+data "vault_generic_secret" "cohcor" {
+  path = "secret/${var.vault_section}/ccidam/service-auth-provider/api/microservice-keys/coh-cor"
```

Comment:
Have you added keys in Hashicorp vault for all environments(test, prepared and prod) ?

Reply:
just waiting on devops to do this

Reply:
ok I have added Do not merge label. Once Devops configures the keys please remove that label so we can approve and merge :)

Reply:
@nitinprabhu there is a concern that the app not even building in jenkins

Reply:
@alectronic0 Not an app problem it is with Jenkins.I have retriggered the build which has now progressed to next stage.

Reply:
ok it passed see build 6 in jenkins (it fails because I removed the suppression which is another PR now)

should be good to merge :+1:  although owasp will fail because  #64 need to be merged too

Reply:
it passing now so should be all good now @nitinprabhu 
:+1: